app/api/services/users.py

```diff
@@ -0,0 +1,32 @@
+from app.api.helpers import Service
+from app import db
+from app.models import User
+from sqlalchemy import func
+
+
+class UsersService(Service):
+    __model__ = User
+
+    def __init__(self, *args, **kwargs):
+        super(UsersService, self).__init__(*args, **kwargs)
+
+    def get_user_organisation(self, domain):
+        """Returns the user's organisation based on their email domain."""
+        query = db.session.execute("""SELECT name FROM govdomains WHERE domain = :domain""", {'domain': domain})
+        results = list(query)
+
+        try:
+            name = results[0].name
+        except IndexError:
+            name = 'Unknown'
+
+        return name
+
+    def get_team_members(self, current_user_id, domain):
+        """Returns a list of the user's team members."""
+        team_ids = db.session.query(User.id).filter(User.id != current_user_id, User.email_address.endswith(domain))
```

Comment:
there can be a potential bug here where it might match more than expected. i.e.
`me.com` can match `xyzme.com`

should also look for the `@` sign while filtering

Reply:
Thanks!  Concatenated the domain with '@'.